handlers/user.go

```diff
@@ -191,9 +192,14 @@ func UserAuthorizeConfirm(w http.ResponseWriter, r *http.Request) {
 	}
 
 	appData, err := db.GetAppDataFromRedirectUri(redirectURI)
-
 	if err != nil {
-		logger.Errorf("error app not found: %v: %v: %v", user.UserID, appData, err)
+		logger.WithFields(
+			logrus.Fields{
+				"user.UserID": user.UserID,
+				"appData":     appData,
+				"redirectURI": redirectURI,
+			},
+		).WithError(err).Errorf("error app not found")
```

Comment:
Why not use `utils.LogError` here too? See the `additionalInfos` parameter.

Reply:
Currently utils.LogError handles the Fields different, we have to merge the other commit first, before using it for good.